main.go

```diff
@@ -64,11 +75,18 @@ func printUsage() {
 	fmt.Println("I'm stupidly manipulating files and directories")
 	fmt.Println("* stupid home")
 	fmt.Println("* stupid cp SRCS DST")
+	fmt.Println("* stupid date [FORMAT]")
 	fmt.Println("* stupid rm SRCS")
 	fmt.Println("* stupid tar SRCS DST")
 	fmt.Println("* stupid untar SRC DST")
 }
 
+func date(format string) error {
+	t := time.Now()
+	fmt.Println(t.Format(format))
```

Comment:
For `stupid home` we used `fmt.Print` to not include a trailing `\n`, we should probably do the same here?